src/DogStatsd.php

```diff
@@ -65,8 +65,8 @@ public function __construct(array $config = array())
         $this->host = isset($config['host']) ? $config['host'] : 'localhost';
         $this->port = isset($config['port']) ? $config['port'] : 8125;
         $this->datadogHost = isset($config['datadog_host']) ? $config['datadog_host'] : 'https://app.datadoghq.com';
-        $this->apiCurlSslVerifyHost = isset($config['curl_ssl_verify_host']) ? $config['curl_ssl_verify_host'] : 2;
-        $this->apiCurlSslVerifyPeer = isset($config['curl_ssl_verify_peer']) ? $config['curl_ssl_verify_peer'] : 1;
```

Comment:
These fields do not exist, and thus, are dynamically declared (with `public` visibility).

However, they are never used.